src/features/campaigns/models/CampaignActivitiesModel.ts

```diff
@@ -102,17 +102,30 @@ export default class CampaignActivitiesModel extends ModelBase {
     if (activitiesFuture.data) {
       const currentActivities = activitiesFuture.data;
 
-      overview.today = currentActivities.filter(
-        (activity) =>
-          (activity.startDate && isSameDate(activity.startDate, todayDate)) ||
-          (activity.endDate && isSameDate(activity.endDate, todayDate))
-      );
-      overview.tomorrow = currentActivities.filter(
-        (activity) =>
-          (activity.startDate &&
-            isSameDate(activity.startDate, tomorrowDate)) ||
-          (activity.endDate && isSameDate(activity.endDate, tomorrowDate))
-      );
+      overview.today = currentActivities.filter((activity) => {
+        if (activity.kind == ACTIVITIES.EVENT) {
+          const startDate = new Date(activity.data.start_time);
+          return isSameDate(startDate, todayDate);
+        } else {
+          return (
+            (activity.startDate && isSameDate(activity.startDate, todayDate)) ||
+            (activity.endDate && isSameDate(activity.endDate, todayDate))
+          );
+        }
```

Comment:
I guess, looking at it again, that this code could have been simplified just a little bit, but this is totally fine IMO.